src/Package/Microsoft.Testing.Platform.DotNetTestClient/Microsoft.Testing.Platform.DotNetTestClient.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    $CommonMetadataElements$
+    <readme>PACKAGE.md</readme>
+    <dependencies>
+      <group targetFramework="net9.0">
```

Comment:
Can't we target net10? It's the target for SDK, right?

Reply:
yes, but testfx does not target it. https://github.com/microsoft/testfx/blob/main/global.json#L3